us_states_game/main.py

```diff
@@ -0,0 +1,32 @@
+import turtle
+import pandas
+
+screen = turtle.Screen()
+screen.title("U.S. States Game")
+image = "blank_states_img.gif"
+screen.addshape(image)
+turtle.shape(image)
+
+data = pandas.read_csv("50_states.csv")
+all_states = data.state.to_list()
+guessed_states = []
+
+while len(guessed_states) < 50:
+    answer_state = screen.textinput(title=f"{len(guessed_states)}/50 States Correct",
+                                    prompt="What's another state's name?").title()
```

Comment:
Here's an interesting error for you:
![image](https://user-images.githubusercontent.com/69209761/206240068-c14e6ed1-e52f-40ba-941d-58f2a18a64d6.png)

Reply:
Im not totally sure why it gets an error. because when I tried to run it it doesn't gets error. The use of that "titile()" case is that even though you type your answer in a lower case or in all caps it still reads your answer the important is to put a str answer.

Reply:
It might be a directory issue -- it's not interfering with functionality, so I'm going to ignore it at this point!